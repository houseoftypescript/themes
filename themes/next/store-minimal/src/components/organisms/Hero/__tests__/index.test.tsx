import hero01 from '@/assets/images/hero/hero01.jpeg';
import { render } from '@testing-library/react';
import { Hero } from '../index';

describe('Hero', () => {
  it('to match snapshot', () => {
    const { container } = render(
      <Hero slides={[{ title: '', image: hero01 }]} />
    );
    expect(container).toMatchSnapshot();
  });
});
