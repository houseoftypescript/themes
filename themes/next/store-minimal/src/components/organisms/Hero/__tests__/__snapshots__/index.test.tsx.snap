// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero to match snapshot 1`] = `
<div>
  <section
    id="hero"
  >
    <div
      class="bg-gray-100"
      style="height: 50vh;"
    >
      <div
        class="relative w-full h-full"
      >
        <div
          class="absolute bg-cover bg-center w-full h-full transition-all duration-500 z-20"
          style="background-image: url(/img.jpg);"
        >
          <div
            class="flex items-center justify-center w-full h-full bg-white/25"
          >
            <div
              class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
            >
              <h1
                class="text-2xl text-center text-gray-800"
              />
            </div>
          </div>
        </div>
        <div
          class="absolute bottom-0 mx-auto w-full z-30"
        >
          <div
            class="flex justify-center py-4 gap-4 items-center"
          >
            <div
              class="bg-cover bg-center w-2 h-2 transition-all duration-500 rounded-full cursor-pointer bg-black"
            />
          </div>
        </div>
      </div>
    </div>
  </section>
</div>
`;
